docs/CNAME

```diff
@@ -0,0 +1 @@
+self.fluidnumerics.com
```

Comment:
This file is needed so that the documentation is hosted at `self.fluidnumerics.com` - it's more that Github references it in order to map the URL we want to use appropriately.